src/com/facebook/buck/junit/TestResult.java

```diff
@@ -50,4 +50,8 @@ public TestResult(String testClassName,
   public boolean isSuccess() {
     return type == ResultType.SUCCESS;
   }
+
+  public String toString() {
```

Comment:
Do you want to check this in? If so, we might want to properly use the stringers to print all fields of the object.